packages/cli-e2e/__tests__/angular.specs.ts

```diff
@@ -459,9 +460,8 @@ describe('ui:create:angular', () => {
     });
 
     it('should run the server on a new port', async () => {
-      await expect(
-        page.goto(tokenServerEndpoint(), {waitUntil: 'load'})
-      ).resolves.not.toThrow();
+      const tokenRequest = await axios.get(tokenServerEndpoint());
+      expect(tokenRequest.data.token).toBeDefined;
```

Comment:
Could go a little bit further to check that it's a string that looks like a JWT token:

```suggestion
      expect(tokenRequest.data.token).toMatch(/^eyJh/)
```